**Define**

Let p =  p(pass) = 0.5 

Let gp = p(good∣pass) = 0.95 

Let gf = p(good∣fail) = 0.75 


**Find** 

p(pass∣good) = ? 

Let ppg = p(pass∣good) = ? 

In [6]:
p = 0.5 

gp = 0.95 

gf = 0.75 


p(pass∣good) = ( p(good∣pass) * p(pass) ) / p(good) 

Solve for p(good)

In [8]:
pg = p * (gp + gf) 
pg

0.85

In [9]:
ppg = (gp * p) / pg
ppg

0.5588235294117647

After applying Bayes' Theorem, we can see that the probability of passing the interview is about 56%
